docs/modules/plugins/partials/proxy-properties.adoc

```diff
@@ -19,8 +25,57 @@ a|`true`
 |`<empty>`
 |Overrides the proxy host which will be passed to browser
 
-|proxy.publish-har-on-failure
+|`proxy.publish-har-on-failure`
 a|`true`
 `false`
 |`false`
 |Enables publishing of attachments with HAR to each failed step. `@noHarOnFailure` meta tag can be used to disable HAR publishing at the story level.
+
+|`proxy.trust-all-servers`
+|`true`
+`false`
+|`false`
+|Whether to disable verification of upstream servers' SSL certificates
+
+|`proxy.capture-types`
+|HAR capture types
+|`REQUEST_CONTENT, RESPONSE_CONTENT, REQUEST_BINARY_CONTENT, RESPONSE_BINARY_CONTENT, REQUEST_HEADERS, RESPONSE_HEADERS`
+|List of HAR capture types
+
+|`proxy.mitm.enabled`
+|`true`
+`false`
+|`false`
+|Whether to enable https://docs.mitmproxy.org/stable[MITM proxy] for the whole test run
+
+|`proxy.mitm.mitm-manager-type`
+|`SELF_SIGNED`
+`IMPERSONATED`
+|`IMPERSONATED`
+|The MITM type using the corresponding certificates
+
+|`proxy.mitm.key-store.type`
+|keystore type
+|`JKS`
+|The https://docs.oracle.com/en/java/javase/11/docs/specs/security/standard-names.html#keystore-types[keystore type]
+
+|`proxy.mitm.key-store.path`
+|path to keystore
+|`<empty>`
+|The relative to `src/main/resources` path to keystore
+
+|`proxy.mitm.key-store.password`
+|password for keystore
+|`<empty>`
+|The password for keystore
+
+|`proxy.mitm.key-store.alias`
+|keystore alias
+|`<empty>`
+|The alias for certificate entry in keystore
+
+|`proxy.mitm.send-certificates`
+|`true`
+`false`
+|`false`
+|Whether to send certificates to servers (only for `SELF_SIGNED` MITM manager type)
```

Comment:
is there a real use-case for such a switch? Maybe you could use some default value for self-signed certs?

Reply:
> is there a real use-case for such a switch? Maybe you could use some default value for self-signed certs?

Yes, indeed, I can hardly imagine a case where user will have to disable the sending of certificates. Using "true" as the default value should be enough